### PR TITLE
Reactifyit

### PR DESCRIPTION
**Pull request by solilokiam**

### CODE REVIEW

File: app/Resources/views/base.html.twig [hunk 1]
```diff
@@ -1,20 +1,36 @@
...
+<body>
+    <div id="react-container">
+        {% spaceless %}
+            {% block body %}
+                {{ react_component('blogApp', {'props': props}) }}
+            {% endblock %}
+        {% endspaceless %}
+    </div>
+{% block footer_scripts %}
+    {% if app.environment == "dev" %}
```
Comment (jorge07) [inline app/Resources/views/base.html.twig:28]: It's a bad practice use environment for features, better add a feature flagging.

File: composer.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     },
     "require": {
         "php": ">=5.3.3",
-        "symfony/symfony": "2.6.*",
+        "symfony/symfony": "2.7.*",
```
Comment (jorge07) [inline composer.json:11]: Thanks!

File: src/Kodify/BlogBundle/Controller/AuthorsController.php [hunk 3]
```diff
@@ -5,53 +5,55 @@
...
+        return new Response($serializer->serialize($author,'json'),200);
+    }
+
+    public function apiAuthorsAction()
+    {
+        $serializer = $this->get('serializer');
+        $authors = $this->getDoctrine()->getRepository('KodifyBlogBundle:Author')->findAll();
 
-        return $this->render('KodifyBlogBundle:Default:create.html.twig', $parameters);
+        return new Response($serializer->serialize($authors,'json'),200);
```
Comment (jorge07) [inline src/Kodify/BlogBundle/Controller/AuthorsController.php]: What about `new JsonResponse`

File: src/Kodify/BlogBundle/Controller/PostsController.php [hunk 4]
```diff
@@ -6,81 +6,106 @@
...
+        $currentPost = $this->getDoctrine()->getRepository('KodifyBlogBundle:Post')->find($id);
+        $serializer = $this->get('serializer');
+
+        if (!$currentPost instanceof Post) {
+            throw $this->createNotFoundException('Post not found');
         }
 
-        // the form element should be passed to the view after validate it to show errors
-        $parameters['form'] = $form->createView();
+        return new Response($serializer->serialize($currentPost,'json'),200);
```
Comment (jorge07) [inline src/Kodify/BlogBundle/Controller/PostsController.php]: same

File: src/Kodify/BlogBundle/Controller/PostsController.php [hunk 5]
```diff
@@ -6,81 +6,106 @@
...
+        $manager = $this->getDoctrine()->getManager();
+        $post = new Post();
+        $post->setTitle($content['title']);
+        $post->setAuthor($manager->getReference('KodifyBlogBundle:Author',$content['author']));
+        $post->setContent($content['content']);
+        $manager->persist($post);
+        $manager->flush();
 
-        return $this->render('KodifyBlogBundle:Default:create.html.twig', $parameters);
+        return new Response($serializer->serialize($post,'json'),200);
```
Comment (jorge07) [inline src/Kodify/BlogBundle/Controller/PostsController.php]: same

File: app/Resources/views/base.html.twig (hunk 1 above)
Comment (solilokiam) [reply app/Resources/views/base.html.twig:28]: it's not a feature, its project tooling config. We only load webpack-dev-server for development env. All other options have to be loaded without webpack

File: composer.json (hunk 2 above)
Comment (solilokiam) [reply composer.json:11]: you welcome. Next one is move it to 3.0

**Comment by jorge07** [comment]
+1

**Pull request merged**